humility-core/src/core/probe.rs

```diff
@@ -96,12 +96,12 @@ impl Core for ProbeCore {
         if let Some(range) = self.unhalted_read.range(..=addr).next_back() {
             if addr + 4 < range.0 + range.1 {
                 let mut core = self.session.core(0)?;
-                return Ok(core.read_word_32(addr)?);
+                return Ok(core.read_word_32(addr.into())?);
             }
         }
 
         self.halt_and_read(|core| {
-            rval = core.read_word_32(addr)?;
+            rval = core.read_word_32(addr.into())?;
```

Comment:
We should probably implement `read_word_64` too.  The default implementation just calls this function twice.